source/docs/hardware/getting-started/can-addressing.rst

```diff
@@ -58,7 +58,9 @@ REV Robotics    5
 Grapple         6
 MindSensors     7
 Team Use        8
-Reserved        9-255
+Reserved        9
+Copperforge     10
```

Comment:
9 should be Kauai Labs. I just updated the doc in ScreenSteps and it makes sense to update here as well.

Reply:
Resolved!